### ISSUE TITLE
App router shows  (SSG)  prerendered as static HTML (uses getStaticProps)

### ISSUE THREAD

**Issue by 23navi** [vercel/next.js#60532]
### Link to the code that reproduces this issue

https://github.com/23navi/Nextjs

### To Reproduce

Create a nextjs app router project. 
Create dynamic page route with generateStaticParams()
Build the page

### Current vs. Expected behavior

It shows  (SSG)     prerendered as static HTML (uses getStaticProps)
App router doesn't have getStaticProps

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 23.2.0: Wed Nov 15 21:53:18 PST 2023; root:xnu-10002.61.3~2/RELEASE_ARM64_T6000
Binaries:
  Node: 20.10.0
  npm: 10.2.3
  Yarn: 1.22.19
  pnpm: 8.7.1
Relevant Packages:
  next: 14.0.4
  eslint-config-next: N/A
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.3.3
Next.js Config:
  output: N/A
```

### Which area(s) are affected? (Select all that apply)

App Router

### Which stage(s) are affected? (Select all that apply)

next build (local)

**Comment by damian-balas** [comment]
I have the same issue :/

**Comment by yiuhoward** [comment]
I saw the same messages when I checked the build logs on Vercel.

I used the App router for generating static content. I don't have the Page router in my project. Does it mean under the hood App router is still using getStaticsProps for SSG?

I'm on `"next": "14.1.3"`

<img width="1199" alt="Screenshot 2024-03-15 at 4 44 38 PM" src="https://github.com/vercel/next.js/assets/17466289/a8297b9d-6587-47ee-bf6c-2c15f511a54d">
